rcl/src/rcl/arguments.c

```diff
@@ -1098,6 +1147,7 @@ rcl_arguments_fini(
       args->impl->num_param_files_args = 0;
       args->impl->parameter_files = NULL;
     }
+    args->impl->allocator.deallocate(args->impl->security_context, args->impl->allocator.state);
```

Comment:
@ivanpauno `rcl_arguments_copy` is lacking code to handle this new member field.

Reply:
Solved in https://github.com/ros2/rcl/pull/515/commits/44a15be05484088c78a4e7cde08a021356757160.